dockerfiles/alpine/is/Dockerfile

```diff
@@ -66,10 +66,12 @@ ARG USER_GROUP_ID=802
 ARG USER_HOME=/home/${USER}
 # build arguments for WSO2 product installation
 ARG WSO2_SERVER_NAME=wso2is
-ARG WSO2_SERVER_VERSION=6.0.0
+ARG WSO2_SERVER_VERSION=6.1.0
 ARG WSO2_SERVER_REPOSITORY=product-is
 ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+# hosted wso2is-6.1.0 distribution URL
```

Comment:
Shall we start comments with a capital letter and end with a fullstop. Look for other places as well

Reply:
Addressed by https://github.com/wso2/docker-is/pull/380